proto/ibc/lightclients/solomachine/v3/solomachine.proto

```diff
@@ -51,8 +48,6 @@ message Header {
 message Misbehaviour {
   option (gogoproto.goproto_getters) = false;
 
-  // ClientID is deprecated
-  string           client_id     = 1 [deprecated = true, (gogoproto.moretags) = "yaml:\"client_id\""];
```

Comment:
It may be useful to `reserve` these fields. cc. @colin-axner 

https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
is this the usage you had in mind @damiannolan 

```
message Misbehaviour {
  option (gogoproto.goproto_getters) = false;

  uint64           sequence      = 2;
  SignatureAndData signature_one = 3 [(gogoproto.moretags) = "yaml:\"signature_one\""];
  SignatureAndData signature_two = 4 [(gogoproto.moretags) = "yaml:\"signature_two\""];

  reserved 1;
}
```

Reply:
hmm, perhaps we should just shift all the field numbers considering its going to be API breaking anyways and require migrations. I think its probably safe to shift them then. Any thoughts or reservations on doing that?

Reply:
I think it should be fine. I don't see any security considerations since this is for providing misbehavior and it isn't stored within the state machine (no migrations are actually needed for this type)